keyboards/arteusx/arteusx.h

```diff
@@ -0,0 +1,4 @@
+#pragma once
+
+#include "quantum.h"
+
```

Comment:
You need to have the layout defined here. 

https://docs.qmk.fm/#/understanding_qmk?id=matrix-to-physical-layout-map